xhtml2pdf/xhtml2pdf_reportlab.py

```diff
@@ -827,9 +827,9 @@ def wrap(self, availWidth, availHeight):
         # IMPORTANT!!! We can not substitute the private value
         # self._colWidths therefore we have to modify list in place
         for i, colWidth in enumerate(newColWidths):
-            if (colWidth is not None) or (colWidth == "*"):
```

Comment:
This second part isn't really an issue, but it is a truism as the first branch will be true already